pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -19,10 +19,22 @@ def __init__(self, options=None):
 
     def set_standard_output_variables(self):
         super().set_standard_output_variables()
-        # Additional standard output variables
+        # Current
+        icell = pybamm.standard_parameters_lithium_ion.current_with_time
```

Comment:
should this be `current_density_with_time`?

Reply:
thanks, should this also be the case for lead acid? since it ends up in the variable "Current density" ?

Reply:
yeah, am expecting `icell` to be a current density for lead-acid as well